bokehjs/src/coffee/models/annotations/arrow.coffee

```diff
@@ -9,8 +9,6 @@ p = require "../../core/properties"
 class ArrowView extends Annotation.View
   initialize: (options) ->
     super(options)
-    if not @model.source?
```

Comment:
was instance of mismatched JS/python defaults (could have been fixed in separate PR but was done here)